nss_simpleauthd/src/ffi.rs

```diff
@@ -0,0 +1,17 @@
+use std::sync::Mutex;
+
+use libnss::group::Group;
+use libnss::interop::Iterator;
+use libnss::passwd::Passwd;
+use libnss::shadow::Shadow;
+
+// Required by nss bindings
+lazy_static::lazy_static! {
+    static ref PASSWD_ITERATOR: Mutex<Iterator<Passwd>> = Mutex::new(Iterator::<Passwd>::new());
+    static ref GROUP_ITERATOR: Mutex<Iterator<Group>> = Mutex::new(Iterator::<Group>::new());
+    static ref SHADOW_ITERATOR: Mutex<Iterator<Shadow>> = Mutex::new(Iterator::<Shadow>::new());
+}
+
+mod group;
```

Comment:
turn this module into a facade: reexport all the module contents here

Reply:
There's nothing to reexport? Unless if all of those `extern "C"` need to be made public to work.

Reply:
good facade. you're right.